tests-system/lobster-json/test_directories.py

```diff
@@ -0,0 +1,71 @@
+import shutil
+
+from .lobsterjsonsystemtestcasebase import LobsterJsonSystemTestCaseBase
+from .lobsterjsonasserter import LobsterJsonAsserter
+
+
+class InputDirectoryJsonTest(LobsterJsonSystemTestCaseBase):
+
+    def setUp(self):
+        super().setUp()
+        self._test_runner = self.create_test_runner()
+
+    def test_missing_directory(self):
+        # lobster-trace: json_req.Input_Not_File_Not_Directory
+        dir_does_not_exist = "directory_does_not_exist"
+        self._test_runner.config_file_data.inputs.append(dir_does_not_exist)
+        self._test_runner.config_file_data.tag_attribute = "directory_test"
+
+        asserter = LobsterJsonAsserter(
+            self,
+            self._test_runner.run_tool_test(),
+            self._test_runner
+        )
```

Comment:
```suggestion
        completed_process = self._test_runner.run_tool_test()
        asserter = LobsterJsonAsserter(self, completed_process, self._test_runner)
```

This could be a better readable format as well as a common format for all the system tests.

Reply:
Done